src/LibFLExBridge-ChorusPluginTests/LibFLExBridge-ChorusPluginTests.csproj

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="GitVersion.MsBuild" Version="5.10.3" PrivateAssets="all" />
     <PackageReference Include="NUnit" Version="3.13.3" />
     <PackageReference Include="SIL.Chorus.LibChorus.TestUtilities" Version="$(ChorusVersion)" />
-    <PackageReference Include="SIL.Chorus.Mercurial" Version="6.5.1.25" IncludeAssets="build" />
+    <PackageReference Include="SIL.Chorus.Mercurial" Version="6.5.1.22" IncludeAssets="build" />
```

Comment:
Why this downgrade? SIL.Chorus.Mercurial 6.5.1.25 fixed running Mercurial on Linux under Python 3, and AFAIK made no other changes. Is there something that I broke in creating 6.5.1.25 that was working correctly with 6.5.1.22 which made the downgrade necessary?